### PR TITLE
Fix 

### PR DESCRIPTION
**Pull request by lmolkova**
Fix Issue https://github.com/Microsoft/ApplicationInsights-dotnet-server/issues/864 .
<Short description of the fix.>

- [x] I ran [Unit Tests](https://github.com/Microsoft/ApplicationInsights-dotnet/blob/develop/.github/CONTRIBUTING.md) locally.

For significant contributions please make sure you have completed the following items:

- [x] CHANGELOG.md updated with one line description of the fix, and a link to the original issue.

Original issue suggests that extra `TelemetryClient.Initialize` call could be removed from the `StartOperation`.
It does not seem to be possible without potentially significant refactoring in `RichPayloadEventSource` and its consumers as it relies on the presence of correlation ids and ikey in the telemetry when it's started.

We also have [documentation and examples](https://docs.microsoft.com/en-us/azure/application-insights/application-insights-custom-operations-tracking) that rely on the fact that correlation ids are set on the telemetery in StartOperation call. It helps users deal with Application Insights telemetery only without knowing anything about Activity.

TL;DR;
While it could be possible to prevent double-initialization (or optimize it to essential properties), it is a potentially breaking change that requires some level of refactoring in the SDK and potentially some consumers. It is a risky change and we should consider doing it when we have strong reasons to.

### CODE REVIEW

File: src/Microsoft.ApplicationInsights/TelemetryClientExtensions.cs [hunk 1]
```diff
@@ -82,6 +82,15 @@ public static IOperationHolder<T> StartOperation<T>(this TelemetryClient telemet
                 throw new ArgumentNullException("operationTelemetry cannot be null.");
             }
 
+            // We intialize telemetry here AND in Track method because of RichPayloadEventSource.
```
Comment (pharring) [inline src/Microsoft.ApplicationInsights/TelemetryClientExtensions.cs]: typo: initialize

File: src/Microsoft.ApplicationInsights/TelemetryClientExtensions.cs [hunk 2]
```diff
@@ -224,6 +233,17 @@ public static IOperationHolder<T> StartOperation<T>(this TelemetryClient telemet
             activity.Start();
             T operationTelemetry = ActivityToTelemetry<T>(activity);
 
+            // We intialize telemetry here AND in Track method because of RichPayloadEventSource.
```
Comment (pharring) [inline src/Microsoft.ApplicationInsights/TelemetryClientExtensions.cs]: typo: initialize

**Comment by lmolkova** [comment]
Also fixes #686
